testing/py-execnet/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=py-execnet
+_pkgname=execnet
+pkgver=1.5.0 
+pkgrel=0
+pkgdesc="execnet: rapid multi-Python deployment"
+url="https://github.com/pytest-dev/execnet"
+arch="noarch"
+license="MIT"
+options="!check" # Tests are outdated
```

Comment:
Can you try workaround from https://github.com/alpinelinux/aports/pull/6059

Reply:
`2 failed, 323 passed, 618 skipped, 6 xfailed in 37.79 seconds`
with diff
```diff
--- a/testing/py-execnet/APKBUILD
+++ b/testing/py-execnet/APKBUILD
@@ -8,9 +8,8 @@ pkgdesc="execnet: rapid multi-Python deployment"
 url="https://github.com/pytest-dev/execnet"
 arch="noarch"
 license="MIT"
-options="!check" # Tests are outdated
 depends="py-apipkg"
-checkdepends="py-six py-tox"
+checkdepends="pytest"
 makedepends="python2-dev python3-dev py-setuptools py-setuptools_scm"
 subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
@@ -24,7 +23,8 @@ build() {
 
 check() {
        cd "$builddir"
-       tox -e py27,py36
+       python2 -m pytest
+       python3 -m pytest
 }
 
```

Reply:
Yep, with new pytest version (that was commited to aports today) results are getting scarier than before. Not even sure how to get thos tests to finish successfully.

Reply:
I reported the other tests
https://github.com/pytest-dev/execnet/issues/89